config/component.config.js

```diff
@@ -62,29 +62,41 @@ export const componentConfig = {
     youtube: {
       color: '#008000',
       model: {
-        url: ''
+        videoId: '',
+        fullscreen: true
       },
       schema: {
         fields: [
           {
             type: 'text',
-            title: 'Url',
-            model: 'url'
+            title: 'Video id',
```

Comment:
Let's change this to 'YouTube video ID". I think, it would be cool to add there some example ID as an input placeholder or under the input, if it's possible